_pages/02_download-dawn.md

```diff
@@ -3,8 +3,8 @@ layout: page
 title: Download DAWN
 permalink: /downloads/
 ---
-### DAWN 2.36.0
 
+### DAWN
```

Comment:
I think we need to let the reader know which version is the latest and available through the links